backend/app/api/endpoints/openapi_responses.py

```
@@ -0,0 +1,550 @@
+        return ResponseObject(
+            id=response_id,
+            created_at=int(datetime.now().timestamp()),
+            status="cancelled",
+            model="unknown",
+            output=[],
+        )
+
+    # Reconstruct model string
+    task_kind = (
+        db.query(Kind)
+        .filter(Kind.id == task_id, Kind.kind == "Task", Kind.is_active == True)
```

Ruff (0.14.8)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)